extensions/fetch/lib.rs

```diff
@@ -589,6 +598,18 @@ pub fn create_http_client(
     builder = builder.proxy(reqwest_proxy);
   }
 
+  if let Some(ncc_l) = no_check_certificate {
+    // Until gets released https://github.com/seanmonstar/reqwest/issues/1210
+    let mut tls = ClientConfig::new();
+    tls.dangerous()
+        .set_certificate_verifier(Arc::new(NoCertificateVerification::new(ncc_l.clone())));
+
+    builder = builder.use_preconfigured_tls(tls)
+
+    // then we might use this one, but will not provide any option to filter which domains are allowed
```

Comment:
The initial idea was to reuse the code but it will not provide fine-grain control over which domains should be excluded from SSL validation.

Reply:
This solution is fine